src/Error/ErrorBuilder.php

```diff
@@ -26,6 +26,7 @@ final class ErrorBuilder
         '/Parameter #1 \$blocks of method Nette\\\\Bridges\\\\ApplicationLatte\\\\SnippetDriver::renderSnippets\(\) expects .*/', # nette internal error
         '/Parameter #3 \$blocks of static method Nette\\\\Bridges\\\\ApplicationLatte\\\\UIRuntime::initialize\(\) expects .*/', # nette internal error
         '/Cannot call method getParent\(\) on Latte\\\\Essential\\\\CachingIterator\|null\./', # latte internal error
+        '/Cannot call method attributes\(\) on Nette\\\\Utils\\\\Html\|null\./', # latte internal error
```

Comment:
I hope this will not hide some real errors

Reply:
Well it might but I do not think it is probable. You usually do not manipulate Html objects in latte tempalte directly